samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
Almost all is possible, but do you really need this? Sample should contain minimal set of "support" code.
My suggestion is to require cmake version to 3.1+ for this sample.